enhancements/update/standalone-worker-surge.md

```diff
@@ -0,0 +1,343 @@
+
+**Cluster Lifecycle Administrator** is a human user responsible for triggering, monitoring, and 
+managing all aspects of a cluster update. They are operating a standalone OpenShift cluster.
+
+1. The cluster lifecycle administrator desires to ensure that there is sufficient worker node capacity during
+   updates to handle graceful termination of pods and rescheduling of workloads.
+2. They want to avoid other causes of drain stalls by limiting the amount of time permitted for any drain operation.
+3. They configure worker `MachineConfigPools` on the cluster with a `MaxSurge`.
+   value that will bring additional worker node capacity online for the duration of an update.
+4. They configure worker `MachineConfigPools` on the cluster with a `NodeDrainTimeout` value of 30 minutes to 
+   limit the amount of time non-capacity related draining issues can stall the overall update.
+
+### API Extensions
+
+#### API Overview
+The Standalone `MachineConfigPool` custom resource is updated to include new update strategies (one of which 
+supports`MaxSurge`) and `NodeDrainTimeout` semantics identical to HCP's `NodePool`.
+
+Documentation for these configuration options can be found in HyperShift's API reference: 
+- https://hypershift-docs.netlify.app/reference/api/#hypershift.openshift.io/v1beta1.NodePoolManagement exposes `MaxSurge`.
+- https://hypershift-docs.netlify.app/reference/api/#hypershift.openshift.io/v1beta1.NodePoolSpec exposes `NodePoolTimeout`
+
+Example `MachineConfigPool` including both `NodeDrainTimeout` and a `MaxSurge` setting:
+```yaml
+kind: MachineConfigPool
+spec:
+  # Existing spec fields are not shown.    
+
+  # Adopted from NodePool to create consistency and further our goal
+  # to improve the reliability of worker updates. This only applies
+  # to drains triggered by the MCO (e.g. CLI triggered drains will
+  # not be impacted).
+  nodeDrainTimeout: 10m
+      
+  # New policy analog to NodePool.NodePoolManagement.
+  machineManagement:
+    upgradeType: "Replace"
+    replace:
+      strategy: "RollingUpdate"
+      rollingUpdate:
+        maxUnavailable: 0
+        maxSurge: 4
+```
+
+Like HCP, `UpgradeType` will support:
+- `InPlace` where no additional nodes are brought online to support draining workloads.
+- `Replace` where new nodes will be brought online with `MaxSurge` support.
+
+#### InPlace Update Type
+
+The `InPlace` update type is similar to `MachineConfigPools` traditional behavior where a 
+user can configure the `MaxUnavailable` nodes. This approach assumes the number (or percentage)
+of nodes specified by `MaxUnavailable` can be drained simultaneously with workloads finding
+sufficient resources on other nodes to avoid stalled drains.
+
+```yaml
+kind: MachineConfigPool
+spec:
+  # Existing spec fields are not shown.    
+
+  machineManagement:
+    upgradeType: "InPlace"
+    inPlace:
+      maxUnavailable: 10%
+```
+
+Nodes are rebooted after they are drained.
+
+#### Replace Update Type

```

Comment:
There's an implicit difference between HCP and MCO in that HCP's replace will always use the latest payload bootimage, and MCO will always use the install-time bootimage. If we do want to get into the business of update types, maybe we should make bootimage management fully explicit as a dependency of this (see linked enhancement above)